src/ristretto/bulletproofs_plus.rs

```diff
@@ -73,10 +74,10 @@ impl TryFrom<&BulletproofsExtendedMask> for RistrettoExtendedMask {
     fn try_from(extended_mask: &BulletproofsExtendedMask) -> Result<Self, Self::Error> {
         let secrets = extended_mask
             .blindings()
-            .map_err(|e| RangeProofError::ExtensionDegree(e.to_string()))?;
+            .map_err(|e| RangeProofError::RPExtensionDegree { reason: e.to_string() })?;
```

Comment:
This abbreviation is not necessary. Rather keep it `ExtensionDegree`. Adding an abbreviation does little to help here